Sources/SwiftMemcache/MemcachedConnection.swift

```diff
@@ -450,7 +534,12 @@ public actor MemcachedConnection {
             _ = try await self.sendRequest(request)
 
         case .finished:
-            throw MemcachedConnectionError.connectionShutdown
+            throw MemcachedError(
+                code: .connectionShutdown,
+                message: "The connection to the Memcached server has shut down.",
+                cause: nil,
+                location: MemcachedError.SourceLocation.here()
```

Comment:
NIT: (Everywhere please)
```suggestion
                location: .here()
```